corehq/apps/data_dictionary/util.py

```diff
@@ -19,10 +19,6 @@
 from corehq.util.quickcache import quickcache
 
 
-class OldExportsEnabledException(Exception):
```

Comment:
I presume you removed this because we never raised this?

Reply:
Yeah, that is correct. I grepped its name, and apart from where we caught it in `generate_data_dictionary.py` it never appears anywhere else in the codebase.